src/features/machine/PolarSettings.js

```diff
@@ -61,14 +65,37 @@ class PolarSettings extends Component {
               </Col>
             </Row>
 
-            <Accordion>
-              <Card className={`${endpointsActiveClass} overflow-auto`}>
-                <Accordion.Toggle as={Card.Header} eventKey={0} onClick={this.props.toggleEndpoints}>
-                  <h3>Force endpoints</h3>
-                  Forces the first and last points to be at the center and edge
-                </Accordion.Toggle>
-              </Card>
-            </Accordion>
+            <Row className="align-items-center pb-2">
+              <Col sm={4}>
+                <Form.Label htmlFor="forceStart">
+                  Start point
+                </Form.Label>
+              </Col>
+
+              <Col sm={8}>
+                <ToggleButtonGroup id="forceStart" type="radio" name="origin" value={this.props.startPoint} onChange={this.props.onStartPointChange}>
+                  <ToggleButton variant="light" value={0}>auto</ToggleButton>
```

Comment:
0,1,2 is fine, but we could easily do more descriptive things, right? Would a string be the right choice, or is there some kind of enum?